live-build/variants/internal-dcenter/ansible/playbook.yml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+- hosts: all
+  connection: chroot
+  gather_facts: no
+  vars:
+    ansible_python_interpreter: /usr/bin/python2
```

Comment:
I think it'd be worth a comment somewhere, stating that dcenter requires python2, and we use python2 to run ansible so that when it runs "pip" it will install the python 2 version of those modules.

Reply:
> dcenter requires python2, and we use python2 to run ansible so that when it runs "pip" it will install the python 2 version of those modules.

If that's the case, I have a slight preference that we install these Python modules through Apt instead and run all Ansible playbooks through Python 3 for consistency. Either that, or create a `.deb` package for DCenter that uses `pip` as part of a separate build system to fetch these modules.